yamux/src/session.rs

```diff
@@ -82,6 +82,9 @@ pub struct Session<T> {
     event_sender: Sender<StreamEvent>,
     // For receive events from sub streams
     event_receiver: Receiver<StreamEvent>,
+    // The control information must be sent successfully and will not cause memory problems
+    unbound_event_sender: UnboundedSender<StreamEvent>,
```

Comment:
`unbounded` is dangerous, giving a large enough bound channel is better.

Reply:
This channel is only used to send control information, there is no possibility of unlimited increase